### ISSUE TITLE
A Gentle Introduction to Dropout for Regularizing Deep Neural Networks

### ISSUE THREAD

**Issue by ikedaosushi** [ikedaosushi/tech-news#9105]
A Gentle Introduction to Dropout for Regularizing Deep Neural Networks<br>
Deep learning neural networks are likely to quickly overfit a training dataset with few examples. Ensembles of neural networks with different model configurations are known to reduce overfitting, but require the additional computational expense of training and maintaining multiple models.<br>
https://ift.tt/2Q7i1PS
